### PR TITLE
1912 Trial Session Tabs and Sorting

### PR DESCRIPTION
**Pull request by matthopson**
Screen Shot 2020-02-04 at 11 10 40 AM

### CODE REVIEW

File: web-client/src/presenter/computeds/formattedTrialSessions.js [hunk 1]
```diff
@@ -20,6 +28,53 @@ export const sessionSorter = sessionList => {
...
+        sessions: [],
+      });
+      termIndex = filtered.length - 1;
+    }
+
+    return termIndex;
+  };
+
+  trialTerms.forEach(trialTerm => {
+    trialTerm.sessions.map(async session => {
```
Comment (rachaelparris) [inline web-client/src/presenter/computeds/formattedTrialSessions.js]: I don't think async is needed here any longer.

File: web-client/src/presenter/computeds/formattedTrialSessions.test.js [hunk 2]
```diff
@@ -21,46 +22,143 @@ const baseState = {
...
+      expect(results.closed.length).toEqual(1);
+      expect(results.closed).toEqual([
+        {
+          ...trialTerms[0],
+          sessions: [sessions[0]],
+        },
+      ]);
+    });
+
+    it('filters open cases', () => {
```
Comment (rachaelparris) [inline web-client/src/presenter/computeds/formattedTrialSessions.test.js]: Probably should be `filters open trial sessions`

File: web-client/src/presenter/computeds/formattedTrialSessions.test.js [hunk 3]
```diff
@@ -21,46 +22,143 @@ const baseState = {
...
+      expect(results.open.length).toEqual(1);
+      expect(results.open).toEqual([
+        {
+          ...trialTerms[0],
+          sessions: [sessions[0]],
+        },
+      ]);
+    });
+
+    it('filters new cases', () => {
```
Comment (rachaelparris) [inline web-client/src/presenter/computeds/formattedTrialSessions.test.js]: Same as above - `filters new trial sessions`

File: web-client/src/presenter/computeds/formattedTrialSessions.test.js [hunk 4]
```diff
@@ -21,46 +22,143 @@ const baseState = {
...
+      expect(results.new.length).toEqual(1);
+      expect(results.new).toEqual([
+        {
+          ...trialTerms[0],
+          sessions: [sessions[0]],
+        },
+      ]);
+    });
+
+    it('filters all cases (returns everything)', () => {
```
Comment (rachaelparris) [inline web-client/src/presenter/computeds/formattedTrialSessions.test.js]: `filters all trial sessions`

**Review by kkoskelin — APPROVED** [review]
👍 once comments are addressed.  Those test labels were my bad!

**Review by rachaelparris — APPROVED** [review]

**Pull request merged**